### ISSUE TITLE
Object element has non-empty accessible name - grammatical error

### ISSUE THREAD

**Issue by moekraft** [act-rules/act-rules.github.io#1409]
Please describe the issue with references and relevant examples where necessary.
https://act-rules.github.io/rules/8fc3b6 
Minor grammatical error found in _Accessibility support._ "Non supported" should be "Non-supported".

--- or ---

If authoring a proposal for a new rule, choose from one of the issue templates:

- [Atomic Rule Template](https://raw.githubusercontent.com/act-rules/act-rules.github.io/develop/pages/design/atomic-template-empty.md)
- [Composite Rule Template](https://raw.githubusercontent.com/act-rules/act-rules.github.io/develop/pages/design/composite-template-empty.md)

**Comment by Jym77** [comment]
@carlosapaduarte I think this is for you.
